Description: Don't set Cflags in .pc
 None of the flags used while compiling the library is needed
 to compile programs _using_ the library.
 .
Author: Balint Reczey <balint@balintreczey.hu>

--- kodiplatform-16.0.0.orig/kodiplatform.pc.in
+++ kodiplatform-16.0.0/kodiplatform.pc.in
@@ -9,4 +9,4 @@ Name: @name@
 Description: @description@ @major@.@minor@
 Version: @major@.@minor@
 Libs: @target@ @libs@
-Cflags: @CMAKE_CXX_FLAGS@
+

